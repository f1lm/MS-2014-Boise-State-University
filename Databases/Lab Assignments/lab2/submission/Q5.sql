-- 5.  Retrieve the names of students whose accumulated credit hours are greater than or equal to 10 
-- (Only passed courses will be counted; A passed course means the student get a grade C or above).
select FName, LName from (
select FName, LName, sum(C.hours) as hours from student S
inner join grade_report G 
on G.SSN = S.SSN 
inner join course C on
G.CourseNo = C.CourseNo
where G.LGrade <= 'C' group by S.SSN) as temp
where hours>=10;