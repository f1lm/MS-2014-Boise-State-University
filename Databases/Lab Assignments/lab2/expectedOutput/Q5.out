mysql> source Q5.sql;
+-------+---------+
| FNAME | LNAME   |
+-------+---------+
| John  | Smith   |
| Jason | Freeman |
+-------+---------+
2 rows in set (0.01 sec)

mysql> notee;
